src/main/cpp/subsystems/SmartIntake.cpp

```diff
@@ -59,43 +59,28 @@ void SmartIntake::HandleInput(RobotControlData& input){
             }
         }
             break;
-        case 2:
-        {
-            input.intakeInput.runIntakeIn = true;
-            if (input.smartIntakeInput.laser){
-                input.intakeInput.runIntakeIn = false;
-                input.intakeInput.goToPseudoStowPos = true;
-                ++m_IntakeState;
-            }
-            
-        }
-            break;
-        case 3:
-        {
-            input.intakeInput.goToPseudoStowPos = false;
-            if (input.intakeOutput.intakePos == IntakePos::PSEUDO_STOW)
-            {
-                ++m_IntakeState;
-            }
-        }
-            break;
+        
         default:
+            m_SmartIntakeFlag = false;
```

Comment:
This should not be in the default case. Make an explicit case for switching over to false. We use the default case to do nothing